packages/toolboxes/evm/src/toolbox/BaseEVMToolbox.ts

```diff
@@ -461,6 +461,51 @@ const sendTransaction = async (
   }
 };
 
+const isAssetValueApproved = (
```

Comment:
instead of having new method that does basically nothing else than isApproved, might want to merge them into one method that takes AssetValue or bigint as param, so we keep code clean